open-craft/opencraft
# New servers should decline new clients by default.

We want to decline new clients for shared services by default. This prevents accidents where we create a new server on Ocim prod which isn't meant to accept new clients just yet, which ends up accepting clients on re-/new provisions of client instances.

Test instructions:

1. Make a new shared server object of some sort on Stage (i.e. MySQL/MongoDB/RabbitMQ/Load Balancer).
2. Ensure it is not accepting new clients/backends by default.
